INSTALL.rst

```diff
@@ -197,6 +211,9 @@ The easiest way to get the project started is by using `Docker Compose`_.
        $ docker exec -it openproduct_web_1 /app/src/manage.py loaddata admin_index groups
        Installed 5 object(s) from 2 fixture(s)
 
+       $ docker exec -it openproduct_web_1 /app/src/manage.py load_upl --url https://standaarden.overheid.nl/owms/oquery/UPL-actueel.csv
```

Comment:
If you use `docker compose exec ...` you can just specify the name of the service in the `docker-compose.yml` file, which is a bit cleaner imo
```suggestion
       $ docker compose exec web /app/src/manage.py load_upl --url https://standaarden.overheid.nl/o
wms/oquery/UPL-actueel.csv
```